client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -27,12 +29,40 @@ import { didOpenResult, setSearchResults } from 'state/inline-help/actions';
 class InlineHelpSearchResults extends Component {
 	static propTypes = {
 		translate: PropTypes.func,
+		searchQuery: PropTypes.string,
 	};
 
 	static defaultProps = {
 		translate: identity,
+		searchQuery: '',
 	};
 
+	state = {
+		railcarSeed: getNewRailcarSeed(),
```

Comment:
Did you & @gibrown discuss the [railcar data coming from the API](https://github.com/Automattic/wp-calypso/pull/22050#discussion_r165756628) thing separately?

I'm still not 💯 sure I understand the seed being something this component generates vs. something it consumes (via props or derived state).